### ISSUE TITLE
pip3 install adafruit-circuitpython-lis3dh

### ISSUE THREAD

**Issue by sUpniverse** [adafruit/Adafruit_CircuitPython_Bundle#117]
i am raspberry user , i want to take picture using vc0706 
in the readme, it said that raspberry user use `pip3 install adafruit-circuitpython-lis3dh` command
so i tried to use command`pip install adafruit-circuitpython-vc0706` and then i faced problem 

the message is `Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-8vbumd7p/Adafruit-GPIO/`

```
Collecting Adafruit-GPIO (from Adafruit-Blinka->adafruit-circuitpython-vc0706)
  Using cached https://files.pythonhosted.org/packages/db/1c/2dc8a674514219f287fa344e44cadfd77b3e2878d6ff602a8c2149b50dd8/Adafruit_GPIO-1.0.3.tar.gz
    Complete output from command python setup.py egg_info:
    Downloading http://pypi.python.org/packages/source/d/distribute/distribute-0.6.14.tar.gz
    Traceback (most recent call last):
      File "/usr/local/lib/python3.4/dist-packages/ez_setup.py", line 143, in use_setuptools
        raise ImportError
    ImportError

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-gi0womhp/Adafruit-GPIO/setup.py", line 4, in <module>
        use_setuptools()
      File "/usr/local/lib/python3.4/dist-packages/ez_setup.py", line 145, in use_setuptools
        return _do_download(version, download_base, to_dir, download_delay)
      File "/usr/local/lib/python3.4/dist-packages/ez_setup.py", line 124, in _do_download
        to_dir, download_delay)
      File "/usr/local/lib/python3.4/dist-packages/ez_setup.py", line 193, in download_setuptools
        src = urlopen(url)
      File "/usr/lib/python3.4/urllib/request.py", line 153, in urlopen
        return opener.open(url, data, timeout)
      File "/usr/lib/python3.4/urllib/request.py", line 461, in open
        response = meth(req, response)
      File "/usr/lib/python3.4/urllib/request.py", line 571, in http_response
        'http', request, response, code, msg, hdrs)
      File "/usr/lib/python3.4/urllib/request.py", line 499, in error
        return self._call_chain(*args)
      File "/usr/lib/python3.4/urllib/request.py", line 433, in _call_chain
        result = func(*args)
      File "/usr/lib/python3.4/urllib/request.py", line 579, in http_error_default
        raise HTTPError(req.full_url, code, msg, hdrs, fp)
    urllib.error.HTTPError: HTTP Error 403: SSL is required

```
 i searched that message and found the a few solution
so i upgraded `setuptool`,`pip`...
but the problem keep occurred 
how can i solve the problem?

**Comment by sUpniverse** [comment]
i already did it...

**Comment by ladyada** [comment]
@sUpniverse did this get resolved? we dont think it has to do with the bundle...but something with your raspberry pi setup

**Comment by sUpniverse** [comment]
Oh i stop using raspberry because  i change  fether m0 board   thx reply : )

**Comment by ladyada** [comment]
ok!

**Comment by dpjenkin** [comment]
@ladyada I'm late to the party here, but I have a similar issue that is driving me crazy. I'm trying to install the Neotrellis library using:

_sudo pip3 install adafruit-circuitpython-neotrellis_

It kicks back with an error like @sUpniverse posted above. Using the tutorial here (https://learn.adafruit.com/adafruit-neotrellis?view=all) it tells me to try using "pip" instead if "pip3" didn't work. I tried "pip" and got an error like "Cannot find a version that satisfies the requirement Adafruit-Blinka"

Any thoughts? I've updated the Pi and rebooted already. I've installed Python 3 and Blinka as well.
